src/ducks/stripeConnectAccount.duck.js

```diff
@@ -154,16 +154,36 @@ export const getAccountLinkSuccess = () => ({
 // ================ Thunks ================ //
 
 export const createStripeAccount = params => (dispatch, getState, sdk) => {
-  const country = params.country;
-  const bankAccountToken = params.bankAccountToken;
+  if (typeof window === 'undefined' || !window.Stripe) {
+    throw new Error('Stripe must be loaded for submitting PayoutPreferences');
+  }
+  const stripe = window.Stripe(config.stripe.publishableKey);
+
+  const { country, accountType, bankAccountToken, businessProfileMCC, businessProfileURL } = params;
+
+  const accountInfo = {
+    business_type: accountType,
+    tos_shown_and_accepted: true,
+  };
 
   dispatch(stripeAccountCreateRequest());
 
-  return sdk.stripeAccount
-    .create(
-      { country, bankAccountToken, requestedCapabilities: ['card_payments', 'transfers'] },
-      { expand: true }
-    )
+  return stripe
+    .createToken('account', accountInfo)
+    .then(response => {
+      const accountToken = response.token.id;
+      return sdk.stripeAccount.create(
+        {
+          country,
+          accountToken,
+          bankAccountToken,
+          requestedCapabilities: ['card_payments', 'transfers'],
```

Comment:
@OtterleyW @sktoiva Could you add some code comments about these capabilities.
Are they mandatory? What they do? (Even a link to Stripe might be a good addition.)